owncloudApp/src/main/res/xml/preferences.xml

```diff
@@ -65,7 +65,7 @@
             android:title="@string/prefs_pattern" />
         <android.preference.CheckBoxPreference
             android:key="set_fingerprint"
```

Comment:
You can modify this to `android:key="set_biometric"` too.

Reply:
I didn't want to modify the sharedpref key so users who have the fingerprint lock enabled don't have to enable it again after updating the apk on PlayStore once this feature gets included. I haven't included it in the new commit, but please, let me know if I should change it.